benchmarks/source-flotiq/scripts/data-update.js

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+
+require(`dotenv`).config();
```

Comment:
Can you please use `NODE_ENV` in the `dotenv` config as done here?

https://github.com/gatsbyjs/gatsby/blob/cf3430462748cecdfb4b9f6b61285534fb4a2ea9/benchmarks/source-contentful/bin/site.js#L7